releasenotes/notes/status-to-object-store.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: environments
+releaseNotes:
+  - |
+    **Added** support for status subresource in Istio Objects.
```

Comment:
This note is not valid.  Status subresources have been supported since 1.6.

Reply:
so no release notes?

Reply:
I'm not sure if this PR needs a note - it's sort of an internal change to enable other work. 

As far as supporting status resources since 1.6... how are we writing to status currently?

Reply:
So this PR doesn't write to the status field, it just lays the groundwork for another PR to write to the status field?

Here is the existing mechanism for writing to the status field.  We need to ensure this work will not overwrite this existing status.  https://istio.io/latest/docs/reference/config/config-status/